homeassistant/components/sensor/__init__.py

```diff
@@ -23,6 +23,7 @@
     DEVICE_CLASS_SIGNAL_STRENGTH,
     DEVICE_CLASS_TEMPERATURE,
     DEVICE_CLASS_TIMESTAMP,
+    DEVICE_CLASS_TOTAL_COST,
```

Comment:
The device class should not tell something about the measurement point's scope. We have state_class for this.

Reply:
Yeah, I agree `DEVICE_CLASS_TOTAL_COST` is not very good, it's an amount of money being measured and it doesn't really matter if that amount is a total cost or something else.